### DISCUSSION TITLE
Units on buGroups setting

### DISCUSSION THREAD

**Discussion by drewj-usnctech** [terrapower/armi#562] [Q&A] [answered]
From docs: https://terrapower.github.io/armi/user/inputs/settings_report.html?highlight=bugroups

![buGroups setting description](https://user-images.githubusercontent.com/83778714/152854017-a3312df7-94a0-4c91-968f-032c1fe386fe.png)

What are the anticipated units on this? Looking at some other examples that only include e.g.,
```yaml
buGroups: [100]
```
my first reaction is %FIMA.

**Comment by john-science** [comment, marked as answer]
Yeah, those are just percentages:

https://github.com/terrapower/armi/blob/75f65b249f1f3afea207c0b6e8a95d1f99144ec7/armi/physics/neutronics/crossSectionGroupManager.py#L652-L659

And the list of percentages has to be monotonic (it can only ever increase).

**Reply by drewj-usnctech** [reply]
Excellent. And just to be pedantic / explicit, its percent FIMA? One could imagine a world where some upper burnup limit is placed in some other units (e.g., MWd/kg HM) and the percentage is in reference to this value. But that would potentially produce > 100% burnup and ruin my lovely hypothetical

**Reply by jakehader** [reply]
Yes you're right that it is %FIMA. This has pretty much been the standard use case at TerraPower, but given that there is a burnup parameter for MWdkg on the blocks (maybe composites too?) then this could be an easy enough extension. I think that there is some interplay with the cross section / lattice physics interface with which parameter is being compared to so maybe we could need to configure an application for which "burnup" is the standard to use.

Consider opening another issue / feature request on this if it is important for your work. Burnup in MWd/kg is clearly a standard in the industry though. Also, if you have time to help with this type of PR it would be appreciated of course! 🙂

**Reply by john-science** [reply]
Yes, it is `%FIMA`.

Perhaps that needs to be in the docstrings somewhere.

**Reply by ntouran** [reply]
yeah we should clarify in the setting definition itself so it shows up on https://terrapower.github.io/armi/user/inputs/settings_report.html?highlight=bugroups

**Reply by john-science** [reply]
@ntouran I will add that to the docs, sure.

**Reply by drewj-usnctech** [reply]
@john-science @jakehader @ntouran thank you for the fast response! I'll make an issue for specifying burnup groups in MWd/kg X. Probably initial heavy metal?
